### ISSUE TITLE
`ExpantaNum.lambertw("ee4992767175317001")` throws an error

### ISSUE THREAD

**Issue by James103** [Naruyoko/ExpantaNum.js#25]
When running:
```javascript
ExpantaNum.lambertw("ee4992767175317001")
```

`ExpantaNum.js` errors out with the following:
```
Uncaught Error: Iteration failed to converge: e1e4992767175317001
d_lambertw@https://naruyoko.github.io/ExpantaNum.js/test.js line 102 > Function:673:11
anonymous/P.lambertw@https://naruyoko.github.io/ExpantaNum.js/test.js line 102 > Function:688:51
anonymous/Q.lambertw@https://naruyoko.github.io/ExpantaNum.js/test.js line 102 > Function:692:30
anonymous@https://naruyoko.github.io/ExpantaNum.js/test.js line 7 > Function:3:19
@https://naruyoko.github.io/ExpantaNum.js/test.js:7:99
```

See also https://github.com/Patashu/break_eternity.js/issues/152.

**Comment by sonic3XE** [comment]
This is because of the attempted iteration becomes Infinity and then NaN for the rest of the iteration.

**Comment by sonic3XE** [comment]
or it might be some precision loss

**Comment by Naruyoko** [comment]
Fix released in [α 1.5.0](https://github.com/Naruyoko/ExpantaNum.js/releases/tag/%CE%B1-1.5.0) and [OmegaNum.js 0.7.0](https://github.com/Naruyoko/OmegaNum.js/releases/tag/v0.7.0)

**Comment by James103** [comment]
> Fix released in [α 1.5.0](https://github.com/Naruyoko/ExpantaNum.js/releases/tag/%CE%B1-1.5.0) and [OmegaNum.js 0.7.0](https://github.com/Naruyoko/OmegaNum.js/releases/tag/v0.7.0)

I can confirm the issue is fixed in both releases.
